src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MEDIUM_TERM = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+
+  let totalCost = days * DAILY_RATE;
```

Comment:
Consider using `const` instead of `let` for `totalCost` since it's not reassigned after the initial calculation. This helps to prevent unintentional reassignments and makes the code more robust.